Gemfile

```diff
@@ -59,6 +59,10 @@ group :development do
   gem "activerecord-deprecated_finders"
 end
 
+group :development, :test do
+  gem 'pry'
```

Comment:
Could we add a `require: false` here? This means that anybody wanting to use pry would have to do `require 'pry'; binding.pry` instead of just `binding.pry` but for the default case of not doing any debugging, I like the lack of additional footprint when running the server or tests.